static/src/js/components/KeywordPicker/KeywordPicker.jsx

```diff
@@ -427,6 +433,7 @@ const KeywordPicker = ({
               <div className="eui-item-list-pane" style={{ marginTop }}>
                 <div className="keyword-picker__search-keywords">
                   <Typeahead
+                    ref={typeaheadRef}
```

Comment:
Should make a test that includes clicking the name in the dropdown. We have one for when a user clicks the Add Keyword button, but not for when a user clicks in the dropdown menu.